include/bout/coordinates.hxx

```diff
@@ -139,101 +139,96 @@ public:
   // Operators
   ///////////////////////////////////////////////////////////
 
-  const Field2D DDX(const Field2D& f, CELL_LOC outloc = CELL_DEFAULT,
-                    const std::string& method = "DEFAULT", REGION region = RGN_NOBNDRY);
-  const Field2D DDX(const Field2D& f, CELL_LOC outloc, DIFF_METHOD method,
-                    REGION region = RGN_NOBNDRY) {
+  Field2D DDX(const Field2D& f, CELL_LOC outloc = CELL_DEFAULT,
+      const std::string& method = "DEFAULT", const std::string& region = "RGN_NOBNDRY");
+  Field2D DDX(const Field2D& f, CELL_LOC outloc, DIFF_METHOD method,
+      const std::string& region = "RGN_NOBNDRY") {
```

Comment:
We still need to provide the `REGION` overloads, just deprecated, before we remove them after 4.3